Makefile

```diff
@@ -0,0 +1,19 @@
+# Use a standard bash shell, avoid zsh or fish
+SHELL:=/bin/bash
+
+# Select the default target, when you are simply running "make"
+.DEFAULT_GOAL:=lint
+
+# local executables
+pip:=./venv/bin/pip
+pre-commit=./venv/bin/pre-commit
+
+.PHONY:lint venv
+
+venv:
+	python3.7 -m venv venv
+	$(pip) install pre-commit
+
+lint:
+	source venv/bin/activate && \
+	pre-commit run --all-files
```

Comment:
Keep it consistent and use `$(pre-commit) run --all-files`

Reply:
It was the intention, arf I should go home at that point !